From 8ed1cf49b10bea7dcc8722e8645683e5422dd053 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 30 Jun 2023 10:25:28 +0200
Subject: [PATCH 0408/1141] arm64: dts: st: add memory rotation on
 stm32mp257f-ev1

Add reserved memory for display rotation of stm32mp257f-ev1 board.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I5e462241f16594ede4044719ddb8809994ae2f6b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/315739
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 9b07cd272ebb..0d2c5a6d5097 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -371,6 +371,7 @@ &ipcc1 {
 };
 
 &ltdc {
+	rotation-memory = <&ltdc_sec_rotation>;
 	status = "okay";
 
 	port {
-- 
2.39.2

